QuantConnect.IBAutomater/StartResult.cs

```diff
@@ -116,6 +121,10 @@ public class StartResult
                     "The two factor authentication request timed out. " +
                     "The request must be confirmed within 3 minutes."
                 },
+                {
+                    ErrorCode.InitializationTimeout,
+                    "The IBAutomater initialization timed out."
```

Comment:
This will now cause Lean to throw and stop right?

Reply:
Correct, previously it only logged the timeout error and Lean continued to attempt connection (which would fail for socket timeout)